docs/content/guides/author-apps/kubernetes/how-to-kubernetes-resource/index.md

```diff
@@ -0,0 +1,118 @@
+---
+type: docs
+title: "How-To: Use a Kubernetes resources in your application"
+linkTitle: "Use a Kubernetes resources in your application"
```

Comment:
This linkTitle is too long. It should not spill over into multi-lines wherever possible. Please match this to similar link titles: 
![image](https://github.com/radius-project/docs/assets/54363786/4a0ab5e9-2fc6-44aa-a081-08c10de843b5)